src/physics.cpp

```diff
@@ -155,10 +155,15 @@ void sample_neutron_reaction(Particle& p)
     p.stream() = STREAM_TRACKING;
   }
 
-  // Play russian roulette if survival biasing is turned on
+  // Play Russian roulette if survival biasing is turned on
   if (settings::survival_biasing) {
-    if (p.wgt() < settings::weight_cutoff) {
-      russian_roulette(p, settings::weight_survive);
+    // if survival normalization is applicable, use normalized weight cutoff and normalized weight survive
+    if ((settings::source_file || settings::surf_source_read)&&(settings::survival_normalization)) {
+      if (p.wgt() < settings::weight_cutoff*p.wgt0()) {
+        russian_roulette(p, settings::weight_survive*p.wgt0());
+      } 
+    } else if (p.wgt() < settings::weight_cutoff) {
+            russian_roulette(p, settings::weight_survive);
```

Comment:
```suggestion
    if (p.wgt() < settings::weight_cutoff*p.wgt0()) {
      russian_roulette(p, settings::weight_survive*p.wgt0());
```
I think we can apply this globally for all cases rather than needing additional settings and checks.